src/main/java/edu/ucsb/ucsbcslas/controllers/CourseController.java

```diff
@@ -28,6 +28,22 @@
 import edu.ucsb.ucsbcslas.models.Course;
 import edu.ucsb.ucsbcslas.repositories.CourseRepository;
 
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.core.env.Environment;
+
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.multipart.MultipartFile;
+import org.springframework.web.servlet.mvc.support.RedirectAttributes;
+
+import com.opencsv.CSVWriter;
+import javax.servlet.http.HttpServletResponse;
+import org.springframework.http.HttpHeaders;
+import java.io.IOException;
+import java.io.StringWriter;
+
```

Comment:
I don't see where we are using any of these imports in this file; I'd remove these if they are not used.

Reply:
fixed, thanks